### ISSUE TITLE
Reduce the size of the npm package by limiting the included files

### ISSUE THREAD

**Issue by paazmaya** [substack/minimist#132]
Looks like the `files` property (https://docs.npmjs.com/files/package.json#files) is not used in `package.json` to specify the included files, nor is the `.npmignore` file (https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) is being used for blacklisting unwanted files, for the package published to npm.

Would you consider adding either the `files` property or the `.npmignore` file, so that the resulting package file would have smaller size?

The current size can be seen when executing the command `npm pack` (https://docs.npmjs.com/cli/pack).

> This issue was create via [tawata](https://github.com/paazmaya/tawata/)

**Comment by paazmaya** [comment]
```sh
$ npm pack
npm notice
npm notice 📦  minimist@1.2.0
npm notice === Tarball Contents ===
npm notice 883B  package.json
npm notice 116B  .travis.yml
npm notice 7.2kB index.js
npm notice 1.1kB LICENSE
npm notice 2.5kB readme.markdown
npm notice 69B   example/parse.js
npm notice 756B  test/all_bool.js
npm notice 3.9kB test/bool.js
npm notice 980B  test/dash.js
npm notice 778B  test/default_bool.js
npm notice 588B  test/dotted.js
npm notice 376B  test/kv_short.js
npm notice 779B  test/long.js
npm notice 909B  test/num.js
npm notice 238B  test/parse_modified.js
npm notice 4.6kB test/parse.js
npm notice 1.6kB test/short.js
npm notice 328B  test/stop_early.js
npm notice 2.5kB test/unknown.js
npm notice 191B  test/whitespace.js
npm notice === Tarball Details ===
npm notice name:          minimist
npm notice version:       1.2.0
npm notice filename:      minimist-1.2.0.tgz
npm notice package size:  7.7 kB
npm notice unpacked size: 30.4 kB
npm notice shasum:        c26c8a845bb1c8c42bfd15259c6df696f30047ac
npm notice integrity:     sha512-N9gLPk92ddIYL[...]iQ16mDCcuUNSA==
npm notice total files:   20
npm notice
minimist-1.2.0.tgz
```

**Comment by paazmaya** [comment]
After adding the `files` property, available via #133:

```sh
$ npm pack
npm notice
npm notice 📦  minimist@1.2.0
npm notice === Tarball Contents ===
npm notice 916B  package.json
npm notice 7.2kB index.js
npm notice 1.1kB LICENSE
npm notice 2.5kB readme.markdown
npm notice === Tarball Details ===
npm notice name:          minimist
npm notice version:       1.2.0
npm notice filename:      minimist-1.2.0.tgz
npm notice package size:  4.0 kB
npm notice unpacked size: 11.7 kB
npm notice shasum:        d48dd8b589aa414a402f03d78be50e3fc190b3b6
npm notice integrity:     sha512-n8LwS8/ltOAQy[...]Tj2eTxyFoy3cg==
npm notice total files:   4
npm notice
minimist-1.2.0.tgz
```

**Comment by LordofMaps** [comment]
This seems like a positive change and low-risk. Could we get this merge in to master?
